viewer/packages/360-images/src/annotation/Image360Annotation.ts

```diff
@@ -19,6 +19,11 @@ export interface Image360Annotation {
    */
   getColor(): Color;
 
+  /**
+   * Get the randomly-generated color originally assigned to this annotation
+   */
+  getOriginalColor(): Color;
+
```

Comment:
This is really confusing, what is the difference between get original color and get color?

Reply:
Conceptually I think it is kind of strange that we assign random color. We don't really do this for anything else. If anything it should at least be an optional thing, and if for some ungodly reason the user wants to override it and then recover it later I think that should be up to the app to cache that color. Would like your opinion on this though.

Reply:
If not then at least call getOriginalColor -> getBaseColor, and (get/set)Color -> (get/set)OverrideColor

Reply:
Alright, decided to remove this  function along with random coloring. From now on, all annotations will have the same color when loaded.